httpx/_api.py

```diff
@@ -18,9 +18,9 @@
     RequestData,
     RequestFiles,
     TimeoutTypes,
-    URLTypes,
     VerifyTypes,
 )
+from ._urls import URL
```

Comment:
could it be under `typing.TYPE_CHECKING`?

Reply:
I'd only use that if it was required to avoid a cyclical dependency.